documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-application-credential-authentication_{context}"]
+= Using application credential authentication with an {osp} source provider
+
+You can use application credential authentication instead of username/password authentication when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
+
+{project-short} supports both of the following types of application credential authentication:
+
+* Application credential ID
+* Application credential name
+
+For each type of application credential authentication, you need to use data from OpenStack to create a `Secret` manifest.
+
+.Prerequisites
+
+Have an {osp} account.
+
+.Procedure
+
+. In the dashboard of the {osp} web console, click *Project > API Access*.
```

Comment:
@RichardHoch 
I think the arrow should not be bold, so "*Project* > *API Access*".